### ISSUE TITLE
Runtime issue

### ISSUE THREAD

**Issue by Alystrasz** [R2Northstar/NorthstarLauncher#267]
I can build `main` branch, but something prevents game from launching (after clicking "Launch Northstar").

![image](https://user-images.githubusercontent.com/11993538/188734760-203997aa-6226-446a-a59a-56cd80340671.png)

[Logs are of no help.](https://pastebin.com/aCrmCFH5)

---

I could build branch `v1.9.4` and use it with no problems.

**Comment by GeckoEidechse** [comment]
Are you building Debug or Release mode?

Also, what happens if you try running the CI build from newest commit?
https://github.com/R2Northstar/NorthstarLauncher/commit/7aa2008ea5a38f66a0b25bb23fa35db6fc2d1d16

**Comment by Alystrasz** [comment]
I tried building and using 7aa2008ea5a38f66a0b25bb23fa35db6fc2d1d16 with both debug and release modes, to no avail; both compile correctly, but after putting dll + exe in game folder, game gets stuck once I click "Launch Northstar".

**Comment by GeckoEidechse** [comment]
What happens if you use the CI build of most recent commit?
https://github.com/R2Northstar/NorthstarLauncher/actions/runs/2982633093

**Comment by Alystrasz** [comment]
Oops, didn't get your message at first :)

Same result while using CI build: 

![image](https://user-images.githubusercontent.com/11993538/188746999-48ef28b4-2019-4973-8af1-a2768ca293d4.png)

**Comment by Alystrasz** [comment]
I asked a friend of mine to build on his computer and send me build results, I used them and got the same result.

I'll check tomorrow if I can identify the breaking commit.

**Comment by GeckoEidechse** [comment]
> I asked a friend of mine to build on his computer and send me build results, I used them and got the same result.
> 
> I'll check tomorrow if I can identify the breaking commit.

Yeah, if anything I believe that unless the issue actually is with some change with Launcher code, the issue is outside of `NorthstarLauncher` and `Northstar.dll`.

**Comment by Alystrasz** [comment]
I tested both using CI builds and building from `main` commits myself, I couldn't get to a stable result: builds can work for a few launches, then stop working with no reason.

Seeing that I can't run https://github.com/R2Northstar/NorthstarLauncher/pull/237 while I successfully reviewed it (on another computer) makes me indeed think of a system-related issue.

I'll check on my desktop computer asap.
